core/src/main/java/org/testcontainers/DockerClientFactory.java

```diff
@@ -40,7 +36,8 @@
     private static final List<DockerConfigurationStrategy> CONFIGURATION_STRATEGIES =
             asList(new EnvironmentAndSystemPropertyConfigurationStrategy(),
                     new DockerMachineConfigurationStrategy(),
-                    new UnixSocketConfigurationStrategy());
+                    new UnixSocketConfigurationStrategy(),
```

Comment:
Please could you switch the order around a bit? I think the least-surprise sequence should be:

- environment and system properties
- Docker Local (for 'native' support)
- Docker Machine
- Unix socket